submarine-server/server-core/src/main/java/org/apache/submarine/server/notebook/NotebookManager.java

```diff
@@ -81,7 +83,17 @@ public Notebook createNotebook(NotebookSpec spec) throws SubmarineRuntimeExcepti
     String lowerName = spec.getMeta().getName().toLowerCase();
     spec.getMeta().setName(lowerName);
     NotebookId notebookId = generateNotebookId();
+
+    Map<String, String> labels = spec.getMeta().getLabels();
+
+    if (labels == null) {
+      labels = new HashMap<>();
+    }
+    labels.put("notebook-owner-id", spec.getMeta().getOwnerId());
+    labels.put("notebookId", notebookId.toString());
```

Comment:
Use "notebook-id" ?
ref: [syntax-and-character-set](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)